generators/generator-botbuilder/generators/app/templates/core/package-with-tests.json.ts

```diff
@@ -38,22 +38,23 @@
     },
     "dependencies": {
       "@microsoft/recognizers-text-data-types-timex-expression": "1.1.4",
-      "botbuilder": "^4.23.1",
-      "botbuilder-ai": "^4.23.1",
-      "botbuilder-dialogs": "^4.23.1",
-      "botbuilder-testing": "^4.23.1",
+      "botbuilder": "4.1.6",
+      "botbuilder-ai": "4.1.6",
+      "botbuilder-dialogs": "4.1.6",
+      "botbuilder-testing": "4.1.6",
       "dotenv": "^8.2.0",
       "replace": "~1.2.0",
       "restify": "~11.1.0"
 },
   "devDependencies": {
       "@types/mocha": "^7.0.2",
+      "@types/node": "^18.19.47",
       "@types/restify": "8.4.2",
       "mocha": "^7.1.2",
       "nodemon": "^2.0.4",
       "nyc": "^15.0.1",
       "ts-node": "^8.10.1",
       "tslint": "^6.1.2",
-      "typescript": "^4.0.7"
+      "typescript": "^4.9.3"
```

Comment:
can we use the latest TS?

Reply:
We can, we just kept the same version as the samples.

Reply:
okay, maybe a different PR to update TS?